src/App.tsx

```diff
@@ -3,16 +3,42 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-type State = {};
+type State = {
+  query: string;
+  movies: Movie[];
+};
 
 export class App extends React.Component<{}, State> {
-  state: State = {};
+  state: State = {
+    query: '',
+    movies: [...moviesFromServer],
+  };
+
+  handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
+    const { query } = this.state;
+
+    this.setState({
+      query: value,
+
+      movies: moviesFromServer.filter(movie => (
+        movie.title.toLowerCase().includes(query.toLowerCase().trim())
+        || movie.description.toLowerCase().includes(query)
```

Comment:
the check should be identical. Now it can't find smth in UpperCase, if it is in a description